src/db.c

```diff
@@ -122,6 +122,46 @@ void updateKeysizesHist(redisDb *db, int didx, uint32_t type, int64_t oldLen, in
     }
 }
 
+/* Assert keysizes histogram (For debugging only)
+ *
+ * To catch commands that corrupt the histogram, you can call it
+ * right after the call to "c->cmd->proc(c);" at server.c */
+void dbgAssertKeysizesHist(redisDb *db) {
```

Comment:
Since it hasn't been used, I tend not to keep it. Can't we simply add it locally?

Reply:
feels more appropriate to add a debug command.

Reply:
pls consider my last commit. thanks.